codes/quantum/qudits_galois/galois_into_galois.yml

```diff
@@ -22,4 +22,4 @@ relations:
 
   cousins:
     - code_id: qudits_into_qudits
-      detail: 'A Galois qudit for \(q=p^m\) can be decomposed into a Kronecker product of \(m\) \(\mathbb{Z}_q\)-qudits \cite{arXiv:quant-ph/0005008}; see Sec. 5.3 of Ref. \cite{arxiv:quant-ph/0501074}. The two coincide when \(q\) is prime, and reduce to qubits when \(q=2\). However, Pauli matrices for the two types of qudits are defined differently.'
+      detail: 'A Galois qudit for \(q=p^m\) can be decomposed into a Kronecker product of \(m\) \(\mathbb{Z}_p\)-qudits \cite{arXiv:quant-ph/0005008}; see Sec. 5.3 of Ref. \cite{arxiv:quant-ph/0501074}. The two coincide when \(q\) is prime, and reduce to qubits when \(q=2\). However, Pauli matrices for the two types of qudits are defined differently.'
```

Comment:
Thanks, I fixed this. Please remove this fix.

Reply:
Done.